src/proxy.js

```diff
@@ -130,6 +131,10 @@ function get(state, prop) {
 		// Store drafts on the copy (when one exists).
 		drafts = state.copy
 	}
+	
+	if (!ownKeys(state.base).includes(prop)) {
```

Comment:
Note that you are checking for enumerability here, not for getter-ness. Those two are in principle unrelated, excepti that getters are by default non-enumerable. 

To find getters and setters you need check `Object.getOwnPropertyDescriptor(object, propname)`, and see if that has a `get` or `set` method.